pkgs/applications/video/mpv/scripts/sponsorblock.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub, fetchpatch, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "mpv_sponsorblock";
+  version = "f71e49e"; # 2020-07-05
+
+  src = fetchFromGitHub {
+    owner = "po5";
+    repo = "mpv_sponsorblock";
+    rev = version;
+    sha256 = "1fr4cagzs26ygxyk8dxqvjw4n85fzv6is6cb1jhr2qnsjg6pa0p8";
+  };
+
+  dontBuild = true;
+
+  patches = [
+    # Use XDG_DATA_HOME and XDG_CACHE_HOME if defined for UID and DB
+    # Necessary to avoid sponsorblock to write in the nix store at runtime.
+    # https://github.com/po5/mpv_sponsorblock/pull/17
+    (fetchpatch {
+      url = "https://github.com/po5/mpv_sponsorblock/pull/17/commits/e65b360a7d03a3430b4829e457a6670b2f617b09.patch";
+      sha256 = "00wv0pvbz0dz2ibka66zhl2jk0pil4pyv6ipjfz37i81q6szyhs5";
+    })
+    (fetchpatch {
+      url = "https://github.com/po5/mpv_sponsorblock/pull/17/commits/3832304d959205e99120a14c0560ed3c37104b08.patch";
+      sha256 = "149ffvn714n2m3mqs8mgrbs24bcr74kqfkx7wyql36ndhm88xd2z";
+    })
+  ];
+
+  postPatch = ''
+    substituteInPlace sponsorblock.lua \
+      --replace "python3" "${python3}/bin/python3" \
+      --replace 'mp.find_config_file("scripts")' "\"$out/share/mpv/scripts\""
+  '';
+
+  # Usage:
+  # pkgs.mpv-with-scripts.override {
```

Comment:
Note that `mpv-with-scripts` is in `aliases.nix`, so it looks to be deprecated. If you look at the git logs for that alias, there used to be a warning thrown when it was used. (That warning was removed for causing ofborg to fail to evaluate)

Also, the current example doesn't evaluate for me: `value is a string with context while a set was expected`

 I believe the correct example here should be:
```
pkgs.wrapMpv pkgs.mpv-unwrapped { scripts = [ pkgs.mpvScripts.sponsorblock ]; }
```
and you should add `passthru.scriptName = "sponsorblock.lua";` to this derivation as well.

Reply:
I get the following error: `error: attribute 'wrapMpv' missing`.
`wrapMpv` doesn't seem to be an exposed package.